packages/terra-badge/tests/nightwatch/BadgeIcon.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import IconBookmark from 'terra-icon/lib/icon/IconBookmark';
```

Comment:
I'm not sure this will work when the site is deployed. I ran into a similar issue with arrange tests. It worked with the webpack dev server, but didn't work when the site was built/deployed. I ended up [mocking the icon code](https://github.com/cerner/terra-core/commit/d34332a48f1f5bdcf9f9b38ba0026fa9387e8916).